### DISCUSSION TITLE
我到底算不算登录了呢

### DISCUSSION THREAD

**Discussion by hk0852** [share121/inter-knot#4936] [灌水]
![屏幕截图 2024-09-16 115611](https://github.com/user-attachments/assets/1637a7af-ade8-4170-9cd4-f58a124ca4c2)

**Comment by yansai0v0** [comment]
一样

**Comment by Minapocalis** [comment]
能发帖应该算是登录吧？

**Comment by share121** [comment]
![image](https://github.com/user-attachments/assets/60ed05fb-99c2-41d4-b5fb-e1393663cf21)
切换到「用户接口」则可登录

**Reply by Bailinghai** [reply]
还是未登录啊
![绳网 2024_9_18 9_59_22](https://github.com/user-attachments/assets/2bef2c26-13c6-43f0-ac8b-d3102949f3e4)

**Reply by Bailinghai** [reply]
退了重新就登上了

**Reply by Bailinghai** [reply]
改回公共又变未登录了-_-||

**Comment by Fluorine-z** [comment]
手机端APP好改吗
